### ISSUE TITLE
NRE when getting OneDrive files in UWP release build

### ISSUE THREAD

**Issue by VitalyKnyazev** [microsoftgraph/msgraph-sdk-dotnet#2025]
**Describe the bug**
UWP app crashes with NRE when built in Release mode, this only happens with latest versions of v4 and v5 packages.

**To Reproduce**
1. Create an UWP app that accesses OneDrive files either via v4.52 or newer (4.51 works ok) or v5.13.0 or newer (5.12.0 works ok).
2. Create a Release build with these AOT settings:
`<Assembly Name="Microsoft.Graph" Serialize="Excluded" Dynamic="Excluded"/>`
4. Launch the app.
5. NRE appears with this exception message "__imp__#15276(...) returned null.", see stack trace below.

**Expected behavior**
Files can be accessed w/o NRE just like in Debug build or in Release build with 4.51 or 5.13.0.

**Desktop (please complete the following information):**
 - OS: Windows 10 build 19045.3208
 - VisualStudio.17.Release/17.6.5+33829.357

**Additional context**
Here is a part of stack trace:
```
Microsoft.Kiota.Serialization.Json.dll!Microsoft.Kiota.Serialization.Json.JsonParseNode.AssignFieldValues<System.__Canon>(System.__Canon item)
Microsoft.Kiota.Serialization.Json.dll!Microsoft.Kiota.Serialization.Json.JsonParseNode.GetObjectValue<System.__Canon>(Microsoft.Kiota.Abstractions.Serialization.ParsableFactory<System.__Canon> factory)
Microsoft.Kiota.Http.HttpClientLibrary.dll!Microsoft.Kiota.Http.HttpClientLibrary.HttpClientRequestAdapter.SendAsync<System.__Canon>(Microsoft.Kiota.Abstractions.RequestInformation requestInfo, Microsoft.Kiota.Abstractions.Serialization.ParsableFactory<System.__Canon> factory, System.Collections.Generic.Dictionary<string, Microsoft.Kiota.Abstractions.Serialization.ParsableFactory<Microsoft.Kiota.Abstractions.Serialization.IParsable>> errorMapping, System.Threading.CancellationToken cancellationToken)
```

**Comment by andrueastman** [comment]
Possibly related to https://github.com/microsoft/kiota-serialization-json-dotnet/pull/86 as the version change of the serialization library is the related change between v5.13.0 and v5.12.0

**Comment by LucaZiegler** [comment]
I have the exact same issue.

**Comment by andrueastman** [comment]
@VitalyKnyazev @LucaZiegler Any chance you can confirm the issue still occurs after the changes made in https://github.com/microsoft/kiota-serialization-json-dotnet/pull/138/files

This should be available in the latest release https://www.nuget.org/packages/Microsoft.Graph/5.32.0

**Comment by VitalyKnyazev** [comment]
I am testing this right now, will provide results soon.

**Comment by VitalyKnyazev** [comment]
> @VitalyKnyazev @LucaZiegler Any chance you can confirm the issue still occurs after the changes made in https://github.com/microsoft/kiota-serialization-json-dotnet/pull/138/files
> 
> This should be available in the latest release https://www.nuget.org/packages/Microsoft.Graph/5.32.0

Thank you, the issue seems to be fixed in 5.32, any plans to migrate it to 4.xx branch?

**Comment by andrueastman** [comment]
Thanks for confirming this @VitalyKnyazev 

Unfortunately, we don't plan on fixing this on the 4.x.x branch as the code generator used in the older version has since been deprecated. We'll close this issue for now.
